### DISCUSSION TITLE
[BLOG • dremio.com] 5 Easy Steps to Migrate an Apache Superset Dashboard to Your Lakehouse | Dremio

### DISCUSSION THREAD

**Discussion by rusackas** [apache/superset#22394] [Show and tell]
[This blog post](https://www.dremio.com/blog/5-easy-steps-to-migrate-an-apache-superset-dashboard-to-your-lakehouse/) was brought up on Slack, and I wanted to relay it here with special thanks to @dipankarqlik

I hope it helps some of you integrate/migrate/troubleshoo/optimize your work with Dremio in your Superset (or Preset) workflows.

https://www.dremio.com/blog/5-easy-steps-to-migrate-an-apache-superset-dashboard-to-your-lakehouse/

**Comment by dipankarmazumdar** [comment]
Thanks @rusackas. And reach out if anyone have any questions around building Superset/Preset dashboards in a Lakehouse architecture & to learn about the benefits!
